src/Pages/Sign/Sign.js

```diff
@@ -1,13 +1,57 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import "./Sign.scss";
 
 class Sign extends Component {
   render() {
     return (
-      <div>
-        
+      <div className="Sign">
+        <div className="pageHeader">
+          <div className="pageTitle">Sign In</div>
+        </div>
+        <div className="formBox">
+          <form className="memberForm">
+            <div className="inputWrapper">
+              <input className="input" placeholder="아이디"></input>
+            </div>
+            <div className="inputWrapper">
+              <input className="input" placeholder="비밀번호"></input>
+            </div>
+            <div className="signInButtonBox">
+              <button className="signInButton">SIGN IN</button>
+              <div className="forgotAccount">
+                <div><a href="#">아이디 찾기</a></div>
```

Comment:
```suggestion
                <div>
                  <a href="#">아이디 찾기</a>
                </div>
```
부모 자식 관계가 잘 드러날 수 있도록 줄바꿈 해주세요!